github-bugzilla-content.js

```diff
@@ -58,7 +58,11 @@ function getPRTitle() {
  * Retrieve the PR url
  */
 function getPRUrl() {
-    return document.URL;
+    let url = document.URL;
+    if (url.indexOf("#") != -1) {
```

Comment:
You can likely simplify into: `return url.replace(/#$/, "");`

Reply:
My bad it's to remove only the final #.
Anyway to do the same: `return url.replace(/#.*/, "");` should be ok.